src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        int number = random.nextInt(100);
```

Comment:
```suggestion
        int number = random.nextInt(BALL_MAX_VALUE);
```
так було трошки більш інформативно